exchanges/stream/mock_test.go

```diff
@@ -0,0 +1,25 @@
+package stream
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/request"
+)
+
+func TestNewMockWebsocketConnection(t *testing.T) {
+	t.Parallel()
+	got := newMockWebsocketConnection()
+	require.NotNil(t, got)
+	require.Panics(t, func() { got.SendMessageReturnResponse(context.Background(), 0, nil, nil) })
+	resp, err := got.SendMessageReturnResponsesWithInspector(context.Background(), 0, nil, nil, 0, nil)
+	require.NoError(t, err)
+	require.Nil(t, resp)
+	singleResp, err := got.SendMessageReturnResponse(request.WithMockResponse(context.Background(), []byte("test")), 0, nil, nil)
+	require.NoError(t, err)
+	require.NotNil(t, singleResp)
+	resp, err = got.SendMessageReturnResponsesWithInspector(request.WithMockResponse(context.Background(), []byte("test")), 0, nil, nil, 0, nil)
+	require.NoError(t, err)
+	require.NotNil(t, resp)
```

Comment:
I think the functionality is important enough to actually check that the result is what you expect. As well as including multi-byte-slice scenario too